modules/machine-config-overview.adoc

```diff
@@ -56,7 +56,7 @@ The kinds of components that MCO can change include:
 * Adding new users by using a machine config is not supported.
 ====
 * **kernelArguments**: Add arguments to the kernel command line when {product-title} nodes boot.
-* **kernelType**: Optionally identify a non-standard kernel to use instead of the standard kernel. Use `realtime` to use the RT kernel (for RAN). This is only supported on select platforms.
+* **kernelType**: Optionally identify a non-standard kernel to use instead of the standard kernel. Use `realtime` to use the RT kernel (for RAN). This is only supported on select platforms. Use the `64k-pages` parameter to enable 64k memory pages for machines in a cluster. This setting is exclusive to machines with 64-bit ARM architectures.
```

Comment:
```suggestion
* **kernelType**: Optionally identify a non-standard kernel to use instead of the standard kernel. Use `realtime` to use the RT kernel (for RAN). This is only supported on select platforms. Use the `64k-pages` parameter to enable the 64k page size kernel. This setting is exclusive to machines with 64-bit ARM architectures.
```

Trying to call out the correct naming, also consistently with the RHEL doc: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_monitoring_and_updating_the_kernel/what-is-kernel-64k_managing-monitoring-and-updating-the-kernel

wdyt?

Reply:
I kept referencing memory page cuz that helped me remember what they were 😆. I like being consistent with the RHEL docs though, updating. Thank you!